Jenkinsfile.central

```diff
@@ -62,13 +62,13 @@ node {
                         --recv-keys ${GPG_KEY_ID} &&
                     echo ${GPG_KEY} | base64 -d >signing_key.asc &&
                     gpg --import signing_key.asc &&
-                    ./sbt ";sonatypeOpen ${BUILD_NUMBER};publish;sonatypeRelease"
+                    ./sbt ';set version in ThisBuild := \\"${params.RELEASE_TAG}\\";publish;sonatypeBundleRelease'
```

Comment:
The 3.x series of sbt-sonatype introduced a new way of handling releases. The concept is to publish to a folder on your local filesystem and then run `sonatypeBundleRelease` to open the staging repository, upload a bundle of the local filesystem, and then cut the release.

See: https://github.com/xerial/sbt-sonatype#buildsbt

Reply:
Should we consider prefixing the `sbt` batch commands here with `sonatypeBundleClean` to ensure that the bundle directory is cleared out across builds?

Reply:
Yeah, I think that makes sense if we're not having Jenkins clean the workspace. I combed through the sbt-sonatype sources and wasn't able to find any evidence that `sonatypeBundleRelease` triggers `sontaypeBundleClean`.